src/state/predictions/index.ts

```diff
@@ -85,6 +109,31 @@ export const predictionsSlice = createSlice({
         newRoundData[futureRound.id] = futureRound
       }
 
+      // For each round, loop through the bets and save them to the bets namespace
+      // if they match the current account
+      if (account) {
+        const betData = rounds.reduce((accum, polledRound) => {
+          const userBet = polledRound.bets.find((bet) => bet.user.address.toLowerCase() === account.toLowerCase())
+
+          if (userBet) {
+            return {
+              ...accum,
+              [polledRound.id]: userBet,
+            }
+          }
+
+          return accum
+        }, {})
+
+        state.bets = {
+          ...state.bets,
+          [account]: {
+            ...state.bets[account],
+            ...betData,
+          },
+        }
+      }
+
```

Comment:
I added this here because we already have 1,000 bets coming each 10 seconds. It would be a shame not to update the `bets` namespace with the relevant data.

Reply:
Disregard this comment.